### PR TITLE
[Term Entry] rfind

### PR DESCRIPTION
**Pull request by roberanegussie**
### Description
I  have added a new tern entry for the string class in python which checks the last occurrence of a substring in index value from a string provided and if the substring is not found it return -1.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/strings/terms/rfind/rfind.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+Title: '.rfind()'
+Description: 'Finds the starting index of the last occurrence of a specified substring if it is found in an original string and if the substring is not present the method returns -1.'
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/rfind/rfind.md:3]:
```suggestion
Description: 'Finds the last occurrence of a specified substring and returns the starting index.'
```

File: content/python/concepts/strings/terms/rfind/rfind.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+Tags:
+  - 'String'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/analyze-data-with-python'
+---
+
+**`.rfind()`** searches an entire string from the `0th` index to the `last` index or searches a part of the string from the `ith` index to `jth` index, where `i < j` and returns the `last` index where the substring is found at else it returns `-1` indicating the substring is not found.
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/rfind/rfind.md:16]:
```suggestion
**`.rfind()`** searches a given string for a substring starting from index `0`,  or alternatively, from the *ith* index to *jth* index (where *i < j*). The method returns the starting index of the last occurrence of the substring.
```

File: content/python/concepts/strings/terms/rfind/rfind.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+str1 = "Hello"
+str2 = "is"
+str3 = "py"
+str4 = "str"
+
+print(mystr.rfind(str1))
+print(mystr.rfind(str2,2,5))
+print(mystr.rfind(str3,0,15))
+print(mystr.rfind(str4,10,20))
+
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/rfind/rfind.md:41]: remove empty line

File: content/python/concepts/strings/terms/rfind/rfind.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+**`.rfind()`** searches an entire string from the `0th` index to the `last` index or searches a part of the string from the `ith` index to `jth` index, where `i < j` and returns the `last` index where the substring is found at else it returns `-1` indicating the substring is not found.
+
+## Syntax
+
+```pseudo
+mystr.rfind(value, start, end)
+```
+
+This method is called on a string `mystr` and returns the last index where the substring `value` is found at by searching the original string `mystr` from the `start` index to the `end` index and if the substring is not found it retuns `-1`.
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/rfind/rfind.md:24]:
```suggestion
This method is called on a string `mystr` and returns the last index where the substring `value` is found at by searching the original string `mystr` from the `start` index to the `end` index, and if the substring is not found it returns `-1`.
```

File: content/python/concepts/strings/terms/rfind/rfind.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+## Example
+
+In the following example, `.rfind()` is used to find out if the substring values are found in the original string `mystr`.
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/rfind/rfind.md:28]:
```suggestion
In the following example, `.rfind()` is used to search for several substring values in the original string `mystr`.
```

File: content/python/concepts/strings/terms/rfind/rfind.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
+---
+Title: '.rfind()'
+Description: 'Finds the starting index of the last occurrence of a specified substring if it is found in an original string and if the substring is not present the method returns -1.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'String'
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/rfind/rfind.md:8]:
```suggestion
  - 'Strings'
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@roberanegussie I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by roberanegussie** [comment]
@caupolicandiaz I've made the requested changes please check.

**Review by caupolicandiaz — APPROVED** [review]
@roberanegussie This is ready for a second review.

**Comment by roberanegussie** [comment]
@caupolicandiaz please merge my PR.

**Comment by caupolicandiaz** [comment]
> @caupolicandiaz please merge my PR.

This entry will be merged after a second review is completed.

**Pull request merged**